packages/clarity-js/types/data.d.ts

```diff
@@ -94,7 +94,8 @@ export const enum Dimension {
     ProductCurrency = 13,
     ProductCondition = 14,
     TabId = 15,
-    PageLanguage = 16
+    PageLanguage = 16,
+    Direction = 17
```

Comment:
Suggest {{Document,/Layout/Text} Direction as Direction is a bit ambiguous (could be pointer direction, et.)